src/ansible/roles/packages/tasks/Fedora.yml

```diff
@@ -45,95 +39,103 @@
       - sudo
       - systemd
       - tcpdump
-      - tig
-      - tmate
-      - tmux
-      - vim
       - wget
       - which
       - wireshark-cli
+      disable_gpg_check: yes
   when: "'base_ground' in group_names"
 
 - name: Install packages for client base image
   block:
+  - name: Install ipa client
+    package:
+      state: present
+      name: '{{ ipa.client }}'
   - name: Install SSSD and its dependencies
-    dnf:
+    package:
       state: present
       name:
-      - freeipa-client
       - nfs-utils
       - realmd
       - sssd
       - sssd-*
   - name: Install debug information for selected packages
-    shell: |
-      dnf debuginfo-install -y  \
-        dbus                    \
-        glibc                   \
-        libcmocka               \
-        libdhash                \
-        libini_config           \
-        libldb                  \
-        libtalloc               \
-        libtevent
-    when: debuginfo
-  when: "'base_client' in group_names"
+    command: dnf debuginfo-install  -y  {{ item }}
```

Comment:
Why this change?

Reply:
Because some of these are not in baseos/appstream and we do not want to taint the machines with pulling things from buildroot.

This way whatever is available is installed and the whole transaction does not fail.

Consider this a direct result of the given constraint to use/include Fedora.yml instead of having a separate standalone RedHat.yaml